Makefile

```diff
@@ -92,8 +98,8 @@ check-tmp:
 prefix-install: configure
 	rm -rf tmp/local tmp/build tmp/site-packages
 	$(PYTHON) setup.py install --prefix="`pwd`/tmp/local" --root=tmp/build
```

Comment:
This makes no sense to me. Why is `pwd` OK here but not the line below?

Reply:
On windows 'pwd' will return something like /c/projects/cysignals/ and that's the right path in this first case.
In the line below you need to remove the '/c/' from the path and `pwd` is evaluated after the $(subst ...). That is why i used `$(CURDIR)` instead.